src/main/java/hudson/plugins/synergy/SynergySCM.java

```diff
@@ -120,37 +120,14 @@ private DescriptorImpl() {
 
 		@Override
 		public boolean configure(StaplerRequest request, JSONObject formData) throws FormException {
-			ccmExe = request.getParameter("synergy.ccmExe");
-			ccmUiLog = request.getParameter("synergy.ccmUiLog");
-			ccmEngLog = request.getParameter("synergy.ccmEngLog");
-			pathName = request.getParameter("synergy.pathName");
+            ccmExe = formData.getString("synergy.ccmExe");

```

Comment:
You should keep the formatting of the source file - as it is using tabs - your code should use tabs.